pages/models/generations/SM1/GENROU/index.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: GENROU
+tags: ["#110","Synchronous machine", "RMS", "EMT", "Phasorial", "Rotor", "Stator", "Dynawo", "Opensource", "GENROU", "GENSAL"]
+date: 10/04/2024 
+last-updated: 23/04/2024
+---
+
+# Salient-pole Synchronous Machine
+
+## Context
+
+As explained in the general Synchronous Machine model, the particular systems that can be derived by making some additional assumptions yield systems of equations that can be easily solved computationally. The GENROU model in *abc* reference frame is one of the simplest models, considering the uniformity of the air gap between the stator and rotor to be able to simplify the equations. 
```

Comment:
can you explain the difference with GENSAL?

Reply:
can you also explain why this simplified model is used in most power system transient stability simulator?

Reply:
GENROU model in abc deleted, as it is the same as the GENSAL when there is no saturattion